.github/workflows/bootstrap.yml

```
@@ -67,13 +67,17 @@ jobs:
         with:
           python-version: "3.12"
       - name: Bootstrap clingo
+        env:
+          SETUP_SCRIPT_EXT: ${{ matrix.runner == 'windows-latest' && 'ps1' || 'sh' }}
+          SETUP_SCRIPT_SOURCE: ${{ matrix.runner == 'windows-latest' && './' || 'source ' }}
+          XP_HOME: ${{ matrix.runner == 'windows-latest' && '$env:userprofile' || '$HOME' }}
```

Just a minor suggestion, otherwise LGTM:
```suggestion
          USER_SCOPE_PARENT_DIR: ${{ matrix.runner == 'windows-latest' && '$env:userprofile' || '$HOME' }}
```

---

Made the suggestion, thanks!